### ISSUE TITLE
[Bug]: Comment is empty string for mods when removed

### ISSUE THREAD

**Issue by aeharding** [LemmyNet/lemmy#5230]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [X] Do you agree to follow the rules in our [Code of Conduct](https://join-lemmy.org/docs/code_of_conduct.html)?
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Summary

When a mod removes a comment, the content of the comment is an empty string (`""`) when viewed by mods/admins.

In older versions of Lemmy, mods and admins could easily see the content of a removed content. This is helpful for determining whether other comments in the chain should be removed, and/or allowing other mods to coordinate further actions (like banning from the community) in a comment thread.

### Steps to Reproduce

1. Remove a comment as a mod
2. View the removed comment as a mod
3. Observe comment content is `""`

### Technical Details

Ansible lemmy install

### Version

0.19.6

### Lemmy Instance URL

lt.harding.dev

**Comment by Nothing4You** [comment]
see #4693 and #4699

**Comment by aeharding** [comment]
Ah, I see. Should this be closed then?

**Comment by dessalines** [comment]
Yes, you can now filter the modlog by post or comment id, so as to quickly view the content and why it was removed. In lemmy-ui we have links that go to these now.
